Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Generic/PlantEnable.mo

```diff
@@ -0,0 +1,395 @@
+﻿within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Generic;
+block PlantEnable "Sequence to enable and disable plant"
+
+  parameter Boolean haveWSE = true
+    "Flag to indicate if the plant has waterside economizer";
+  parameter Real schTab[4,2] = [0,1; 6*3600,1; 19*3600,1; 24*3600,1]
+    "Plant enabling schedule allowing operators to lock out the plant during off-hour";
+  parameter Modelica.SIunits.Temperature TChiLocOut=277.5
+    "Outdoor air lockout temperature below which the chiller plant should be disabled";
```

Comment:
Though according to the sequence the plant can operate until oat<lockoutT - 1, which I think they should edit in the document (move that essentially hysteresis offset 1 degree up).